src/integration-test/java/io/aiven/kafka/connect/opensearch/OpensearchClientIT.java

```diff
@@ -80,7 +80,7 @@ void createIndexDoesNotCreateWhenAliasExists() throws Exception {
 
         try {
             tmpClient.client.indices().create(
-                new CreateIndexRequest("index_4").alias(new Alias("alias_1")),
+                new CreateIndexRequest("index_6").alias(new Alias("alias_1")),
```

Comment:
didn't get this change could you please elaborate?

Reply:
yeah, we already create `index_4` in different test case :-)

Reply:
hmmm..
what do you think about usage of _org.junit.jupiter.api.TestInfo_ like
```java
@Test
    void createIndexDoesNotCreateWhenAliasExists(TestInfo testInfo) throws Exception {
...
new CreateIndexRequest(testInfo.getTestMethod().get().getName()).alias(new Alias("alias_1")),
...
```
Since method names are unique within a class you don't need to remember about this counter
thoughts?

Reply:
makes sense
may be later i'll come with a pr for that